File: RepoTransBench/target_projects/Java/C#/woodpecker-framework_ysoserial-for-woodpecker/public_tests/YsoConfigSmokePublicTest.cs
using Xunit;

namespace WoodpeckerYsoserial.PublicTests
{
    public class YsoConfigSmokePublicTest
    {
        [Fact]
        public void TestDefaultCompressFlag()
        {
            var config = new YsoConfig();
            Assert.False(config.IsCompress());
        }

        [Fact]
        public void TestSetCompressToTrue()
        {
            var config = new YsoConfig();
            config.SetCompress(true);
            Assert.True(config.IsCompress());
        }

        [Fact]
        public void TestSetCompressToFalse()
        {
            var config = new YsoConfig();
            config.SetCompress(true);
            config.SetCompress(false);
            Assert.False(config.IsCompress());
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/woodpecker-framework_ysoserial-for-woodpecker/public_tests/yso_config_smoke_public_test.go
package public_tests

import (
	"testing"

	"github.com/stretchr/testify/assert"
	yso "woodpecker-framework-ysoserial"
)

func TestYsoConfigSmokePublic_DefaultCompressFlag(t *testing.T) {
	conf := yso.NewYsoConfig()
	assert.False(t, conf.IsCompress())
}

func TestYsoConfigSmokePublic_SetCompressToTrue(t *testing.T) {
	conf := yso.NewYsoConfig()
	conf.SetCompress(true)
	assert.True(t, conf.IsCompress())
}

func TestYsoConfigSmokePublic_SetCompressToFalse(t *testing.T) {
	conf := yso.NewYsoConfig()
	conf.SetCompress(true)
	conf.SetCompress(false)
	assert.False(t, conf.IsCompress())
}